src/python/grpcio_tests/tests/unit/_interceptor_test.py

```diff
@@ -70,9 +72,13 @@ def handle_unary_unary(self, request, servicer_context):
                 'testkey',
                 'testvalue',
             ),))
+        if request == _EXCEPTION_REQUEST:
+            raise RuntimeError()
```

Comment:
It might be helpful to raise a special exception private to this test case, so the test case can deterministic catch it. In case, we somehow perform some change that causes `RuntimeError`.